content-style-guide/01-voice-and-tone.md

```diff
@@ -0,0 +1,98 @@
+
+## Our voice
+
+Node.js’ voice is relatable. It’s friendly, encouraging, and clear. Our priority
+is explaining Node.js and helping our users do what they need with it. We want to
+educate anyone using our content. We don't want to patronize or confuse them.
+
+Node.js’ voice is:
+
+* **Helpful**
+* **Respectful**
+* **Expert**
+* **Inviting**
+
+### Examples of voice
+
+The Node.js [Guides](https://nodejs.org/en/docs/guides/) is one section where content
+needs to be instructive. But being instructive should not extend to being bossy. Instead consider inviting your reader to follow the steps you are detailing.
+
+You might say: _"If you'd like to give this a go..."_
+
+Instead of: _"Try it out."_
+
+Formality can be perceived as stuffy or aloof. The Node.js [Get Involved](https://nodejs.org/en/get-involved/) section is one place where we want
+to be personal and inviting. To achieve this, consider addressing your reader like
+a friend.
+
+You might say: _"You're welcome to chat with us about Node.js core features on our
+Github issues list!"_
+
+Instead of: _"The GitHub issues list is the place for discussion of Node.js core
+features."_
+
+## Our tone
+
+Node.js’ tone is usually informal. When writing content or a response, being clear
+and direct is always more important than being entertaining.
+
+Actions are also part of tone. A request for help going ignored for many weeks could be perceived as unwanted and unworthy. A quick response, even if an issue itself remains unresolved, shows compassion and thoughtfulness.
+
+It will help to know the audience and the situation when deciding on the right tone. When writing for an audience, think about asking yourself these six questions:
+
+* Who will be reading this content or post? _This is your intended audience._
+* What does the reader want? _This is their goal._
+* How familar is the reader with the content? _This lets you tailor your answer
+to them._
+* What situation is the reader in? _This helps you understand why they are here._
+* What emotion is the reader feeling at this moment? _This helps you empathise,
+to help better._
+* How can you maintain (or better yet, improve) the reader's state of mind? _This
+is your goal._
+
+When you understand the context to which the conversation belongs, you can adjust
+your tone to fit.
+
+### Examples of tone
+
+The [Issues](https://github.com/nodejs/help/issues) tracker of the Node.js Help
+repo  is an example of an area where tone is especially important. Users of Node.js
+are opening issues to solicit help. It's often difficult to ask for help! You can
+make their experience directly interacting with Node.js a good one by putting thought
+into using a tone appropriate to the request.
+
+They might say: _"How do I load another .js file and run that new .js file in a new
+global context?..."_
+
+You might respond: _"If you really want to do this, you will need to put a wrapper
```

Comment:
imo `if you really want to do this` should not be said here, rather, it should be explained immediately _why_ you shouldn't do this